hardware/ASyncTCP.h

```diff
@@ -1,11 +1,15 @@
 #pragma once
 
 #include <stddef.h>                        // for size_t
+#include <queue>						   // for write queue
 #include <boost/asio/deadline_timer.hpp>   // for deadline_timer
 #include <boost/asio/io_service.hpp>       // for io_service
 #include <boost/asio/ip/tcp.hpp>           // for tcp, tcp::endpoint, tcp::s...
+#include <boost/asio/ssl.hpp>			   // for secure sockets
```

Comment:
I think you have to use the preprocessor 'WWW_ENABLE_SSL' also in this class (openssl has to be found correct?, see webserver/server.hpp)

Reply:
Doine